react-app/src/Finish/index.js

```diff
@@ -4,8 +4,37 @@ import Header from './Header'
 import Image from './Image'
 import Buttons from './Buttons'
 import Lottie from 'lottie-react-native'
+import { useEffect, useState } from 'react'
+import * as SQLite from 'expo-sqlite'
 
 export default function Finish() {
+  const db = SQLite.openDatabase('shelter.db')
```

Comment:
I think we've already had a database object `db` which is from prop.

Reply:
Fixed!